hack/run-ci-e2e-test.sh

```diff
@@ -74,13 +74,19 @@ KEY_PAIR_NAME=${KEY_PAIR_NAME:-"openshift-dev"}
 OPERATOR_IMAGE=${OPERATOR_IMAGE:-${IMAGE_FORMAT//\/stable:\$\{component\}//stable:windows-machine-config-operator-test}}
 
 # Setup and run the operator
-run_WMCO $OSDK
+if ! run_WMCO $OSDK; then
+  cleanup_WMCO $OSDK
+  exit 1
+fi
 
 # The bool flags in golang does not respect key value pattern. They follow -flag=x pattern.
 # -flag x is allowed for non-boolean flags only(https://golang.org/pkg/flag/)
 # Run the creation tests and skip deletion of the Windows VMs
 OSDK_WMCO_test $OSDK "-run=TestWMCO/create -v -timeout=90m -node-count=$NODE_COUNT -skip-node-deletion -ssh-key-pair=$KEY_PAIR_NAME"
 
+# Get logs on success before running the destroy tests
+get_WMCO_logs
```

Comment:
This will result in the logs being gathered twice, once here, and then once on either success or failure of the deletion suite. Since the operator pod is the same between the creation and deletion suites the logs will have duplicate information.

Reply:
Good point. I added this because I could not figure out why all logs were not being shown but I think the main issue was with `get_WMCO_logs` itself which I fixed by adding the `-tail` option.